GitSearch/Models/SpinnerViewController.swift

```diff
@@ -24,3 +24,20 @@ class SpinnerViewController: UIViewController {
             ])
     }
 }
+
+extension SpinnerViewController {
+    
+    func showHide(parent: UIViewController, _ show: Bool = true) {
```

Comment:
Thats not a best name for a function. I would recommend to do small refactoring here:
1. Create separate methods `add`/`remove`
2. Create wrapper method, `switch` or `toggle` or something similar which will accept `Bool` and call either `add` or `remove`.

Reply:
changed `init` way, simpled toggle operation
I made nested functions `add/remove`, but they also can be as private extension outside `toggle` func.